plugins/Lv2Instrument/Lv2Instrument.cpp

```diff
@@ -257,35 +256,45 @@ Lv2InsView::Lv2InsView(Lv2Instrument *_instrument, QWidget *_parent) :
 
 void Lv2InsView::dragEnterEvent(QDragEnterEvent *_dee)
 {
-	// For mimeType() and MimeType enum class
-	using namespace Clipboard;
-
-	void (QDragEnterEvent::*reaction)() = &QDragEnterEvent::ignore;
-
-	if (_dee->mimeData()->hasFormat( mimeType( MimeType::StringPair )))
+	const QMimeData* mime = _dee->mimeData();
+	if (mime->hasUrls())
 	{
-		const QString txt =
-			_dee->mimeData()->data( mimeType( MimeType::StringPair ) );
-		if (txt.section(':', 0, 0) == "pluginpresetfile") {
-			reaction = &QDragEnterEvent::acceptProposedAction;
+		const QList<QUrl> urls = mime->urls();
+		if (!urls.isEmpty())
```

Comment:
```suggestion
		if (urls.isEmpty() == false)
```
Also could you do this for all of these? I think using "!" may make the code harder to read than "== false"